src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessToken/AccessTokenHandler.cs

```diff
@@ -118,7 +120,8 @@ private async Task<bool> ValidateAccessToken(string token)
             SecurityToken validatedToken;
```

Comment:
There are logged a few warnings in this file that are really annoying me in the logs..
Could you remove log.Warning informing that token is being validated and is successfully validated? 
Definetly worth pushing a new nuget as they fill up our logs.

Reply:
removed